src/os/src/detail/osTime.c

```diff
@@ -565,15 +565,23 @@ int64_t taosTimeSub(int64_t t, int64_t duration, char unit, int32_t precision) {
   return (int64_t)(mktime(&tm) * TSDB_TICK_PER_SECOND(precision));
 }
 
-
 int32_t taosTimeCountInterval(int64_t skey, int64_t ekey, int64_t interval, char unit, int32_t precision) {
```

Comment:
The taosTimeCountInterval  is to count the number of aligned local time intervals, which are involved by the interval [skey, ekey] inclusively.

Reply:
The taosTimeCountInterval function is to count the number of aligned local time intervals to cover the closed range of time [skey, ekey].